python/ray/tune/search/ax/ax_search.py

```diff
@@ -217,12 +219,15 @@ def _setup_experiment(self):
                     "Please specify the `mode` argument when initializing "
                     "the `AxSearch` object or pass it to `tune.TuneConfig()`."
                 )
+
+
             self._ax.create_experiment(
                 parameters=self._space,
-                objective_name=self._metric,
+                objectives={
+                    self._metric: ObjectiveProperties(minimize=self._mode != "max")
+                },
```

Comment:
Could you make a note of what version of ax this requires?

Want to confirm it works with what's currently pinned [here](https://github.com/ray-project/ray/blob/3d830fb924502f30f286a8f8e6980e400a9fa93a/python/requirements/ml/tune-requirements.txt#L2) (0.3.2)